dist/unshimmed-base.txt

```diff
@@ -0,0 +1,26 @@
+com/nvidia/spark/SQLPlugin*
+com/nvidia/spark/RapidsUDF*
+com/nvidia/spark/rapids/ExecutionPlanCaptureCallback*
+com/nvidia/spark/rapids/PlanUtils*
+com/nvidia/spark/rapids/SQLExecPlugin*
+com/nvidia/spark/rapids/ShimLoader*
+com/nvidia/spark/rapids/SparkShimServiceProvider*
+com/nvidia/spark/rapids/SparkShims*
+com/nvidia/spark/rapids/ShimVersion*
+com/nvidia/spark/rapids/SparkShimVersion*
+com/nvidia/spark/rapids/RapidsExecutorHeartbeatMsg*
+com/nvidia/spark/rapids/RapidsExecutorStartupMsg*
+com/nvidia/spark/rapids/RapidsExecutorUpdateMsg*
+org/apache/spark/sql/rapids/VisibleShuffleManager*
+org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase*
```

Comment:
I will rework this in a follow up after fixing the regression from prototype @tgravescs found. 
The general idea is that only the classes on the class load path from Spark are "unshimmed".  RapidsShuffleInternalManager* is wrapped into a Proxy that is loading it via ShimLoader from the plugin. So I need to make sure just like in the prototype it's not referenced directly in bytecode